parbase/FairParAsciiFileIo.h

```diff
@@ -23,20 +23,35 @@ class FairParAsciiFileIo : public FairParIo
   public:
     FairParAsciiFileIo();
 
-    // default destructor closes an open file and deletes list of I/Os
+    /**
+     * default destructor closes an open file and deletes list of I/Os
+     */
     ~FairParAsciiFileIo();
```

Comment:
While here, could you add a `override`?

```suggestion
    ~FairParAsciiFileIo() override;
```